pkg/proxy/nftables/proxier.go

```diff
@@ -601,6 +606,15 @@ func (proxier *Proxier) setupNFTables(tx *knftables.Transaction) {
 			),
 		})
 	}
+	if proxier.serviceCIDRs.Len() > 0 {
+		tx.Add(&knftables.Rule{
+			Chain: kubeServicesChain,
+			Rule: knftables.Concat(
+				ipX, "daddr", "==", "{", strings.Join(proxier.serviceCIDRs.UnsortedList(), ","), "}",
```

Comment:
"==" is optional in rules, and the "canonical" syntax is to omit it:

```
$ nft add chain ip test equals
$ nft add rule ip test equals 'ip daddr == { 1.2.3.4, 5.6.7.8 } drop'
$ nft list chain ip test equals
table ip test {
	chain equals {
		ip daddr { 1.2.3.4, 5.6.7.8 } drop
	}
}
```

Reply:
This might be a good example of a rule that should have a comment explaining it...
(nftables comment, not a golang comment)